wandb/sdk/lib/disabled.py

```diff
@@ -11,3 +11,9 @@ def __getitem__(self, key):
             val = SummaryDisabled(val)
             self[key] = val
         return val
+
+
+class RunDisabled:
```

Comment:
Do we need to add the methods on it? maybe we can subclass Run? So all methods would be there?

Reply:
I'm worried about circular imports and also just want to limit its footprint to be able to remove it once all the references to this class in integrations are gone

Reply:
added a deprecation message instead